src/drive/web/modules/views/Drive/index.jsx

```diff
@@ -75,6 +88,9 @@ const DriveView = ({
   displayedFolder
 }) => {
   useTrashRedirect(displayedFolder)
+  const { isMobile } = useBreakpoints()
+
+  const showAdditionalColumns = !isMobile && __TARGET__ !== 'mobile'
```

Comment:
Again, we have business logic on the columns here, let's have the source of truth once.